pages.id/common/git-pull.md

```diff
@@ -0,0 +1,16 @@
+# git pull
+
+> Tarik cabang dari repositori remote and menggabungkan ke repositori lokal
+> Informasi lebih lanjut: <https://git-scm.com/docs/git-pull>.
+
+- Unduh perubahan dari bawaan repositori remote dan menggabungkannya:
+
+`git pull`
+
+- Unduh perubahan dari bawaan repositori remote dan menggunakan maju cepat:
+
+`git pull --rebase`
+
+- Unduh perubahan dari repositori remote dan cabang yang diberikan, kemudian menggabungkannya ke KEPALA:
```

Comment:
```suggestion
- Unduh perubahan dari repositori remote dan cabang yang diberikan, kemudian menggabungkannya ke HEAD:
```
Oh wait can you revert this to HEAD (since this is Git's own terms)?

Reply:
Agreed, Leaving it as it as the main page would be better.(HEAD)

Reply:
Suggestion addressed. I was thinking about the same thing before. Thanks!